pkg/metadata/main_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+)
+
+func TestWriteConfig(t *testing.T) {
+	basePath, err := ioutil.TempDir(os.TempDir(), "metadata")
```

Comment:
Passing `""` (the empty string) as the first argument has the same effect as manually passing `os.TempDir()` but is more usual.